index.bs

```diff
@@ -257,6 +264,50 @@ for [=user verification=], along with appropriate driver software to mediate acc
 authenticators MAY operate autonomously from the [=client device=] running the user agent, and be accessed over a transport such
 as Universal Serial Bus (USB), Bluetooth Low Energy (BLE) or Near Field Communications (NFC).
 
+## Specification Roadmap ## {#spec-roadmap}
+
+While many W3C specifications are directed primarily to user agent developers and also to web application developers
+(i.e., "Web authors"), the nature of Web Authentication requires that this specification be correctly used by multiple audiences,
+as listed below.
+All audiences ought to begin with [[#use-cases]], [[#sample-scenarios]], and [[#terminology]], and should also
+refer to [[WebAuthnAPIGuide]] for an overall tutorial.
+<br><br>
+- [=[RP]=] web application developers, expecially those responsible for [=[RP]=] [=web application=] login flows, account recovery flows,
+    user account database content, etc.
+- Web framework developers
+
+    - The above two audiences should in particular refer to [[#rp-operations]].
+        The introduction to [[#api]] may be helpful, though readers should realize that the [[#api]] section is targeted specifically
+        at user agent developers, not web application defelopers.
+        Additionally, if they intend to verify [=authenticator=] [=attestations=], then
+        [[#sctn-attestation]] and [[#defined-attestation-formats]] will also be relevant.
+        [[#extensions]], and [[#sctn-defined-extensions]] will be of interest if they wish to make use of extensions.
+        <br><br>
+
+- User agent developers
+- OS platform developers, responsible for OS platform API design and implementation in regards to platform-specific
+    [=authenticator=] APIs, platform [=WebAuthn Client=] instantiation, etc.
+
+    - The above two audiences should read [[#api]] very carefully, along with [[#extensions]] if they intend to support extensions.
+        <br><br>
+
+- [=Authenticator=] developers. These readers will want to pay particular attention to [[#sctn-authenticator-model]],
+    [[#defined-attestation-formats]], [[#extensions]], and [[#sctn-defined-extensions]].
+    <br><br>
+
+<div class="note">
+    Note: Along with the [[#api|Web Authentication API]] itself, this specification defines a 
+    request-response <em>cryptographic protocol</em>
+    between a [=[WRP]=] server and an [=authenticator=], comprised of 
+    {{PublicKeyCredentialCreationOptions|input data}} supplied by the [=[RP]=] and a digitally signed
+    [=authenticator data=] message emitted by the [=authenticator=] in reply. The latter protocol is in turn conveyed via the
```

Comment:
It's not clear to me what "the latter protocol" is meant to refer to. Perhaps this could be said as "This reply is in turn..."?

Reply:
it's referring to the cryptographic Authnr <--> RP protocol, described in the prior sentence.  I've re-written that parag to attempt to clarify.